hw_24/tests/test_contacts.py

```diff
@@ -0,0 +1,270 @@
+"""Homework_24"""
+
+
+import time
+import random
+import string
+import pytest
+from selenium import webdriver
+from selenium.common import TimeoutException
+from selenium.webdriver.chrome.service import Service
+from selenium.webdriver.common.by import By
+from selenium.webdriver.chrome.options import Options
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from webdriver_manager.chrome import ChromeDriverManager
+from hw_24.source.locators import (AddUserLocators, LoginLocators,
+                                   LogoutLocators, AddContactLocators,
+                                   UpdateContactLocators, DeleteContactLocators)
+
+
+@pytest.fixture(scope="module")
+def driver():
+    chrome_options = Options()
+    chrome_options.add_argument('--start-maximized')
+    service = Service(ChromeDriverManager().install())
+    driver = webdriver.Chrome(service=service, options=chrome_options)
+    driver.implicitly_wait(10)
+    yield driver
+    driver.quit()
+
+
+def random_char(char_num):
+    return ''.join(random.choice(string.ascii_letters) for _ in range(char_num))
+
+
+EMAIL = random_char(7) + '@test.com'
+PASSWORD = random_char(11)
+
+
+def test_signup(driver):
```

Comment:
you gave this method the name `test_*`, but it doesn't test anything =(

Reply:
fixed this one and test_login